rts/Lua/LuaOpenGL.cpp

```diff
@@ -5110,6 +5168,71 @@ int LuaOpenGL::ReadPixels(lua_State* L)
 }
 
 
+namespace Impl {
+	template<class Type>
+	static inline void ReadAttachmentPixel(lua_State* L, GLint x, GLint y, GLenum format, GLenum readType, int fSize) {
+		Type data[fSize];
+		glReadPixels(x, y, 1, 1, format, readType, data);
+		for (int e = 0; e < fSize; e++) {
+			lua_pushnumber(L, data[e]);
+		}
+	}
+}
+
+int LuaOpenGL::ReadAttachmentPixel(lua_State* L)
```

Comment:
Documentation would be good. https://github.com/beyond-all-reason/spring/pull/935/files#r1281757982 says that this is rather for development (editor/debug) and can have low perf, which are important caveats